Feature/num 511 integration test vaccination

af-dpe opened the pull request:
added jsons for imunization "completed" and "not done"
added location of the jsons and robot file for imunization to the suite setting file
added immunization robot
- keyword for validate
- keyword for create
- keywords for create
added 01_create in Immunization folder with Testcases
- create History of vaccination - immunization completed
- create History of vaccination - immunization not done
added __init__.robot to folder immunization


**Currently the ressourceType - Immunization is not accepted and the test failes with 404**

StefanWinterfeldtVitasystems reviewed (commented):
Could the Maven Wrapper also be downloaded and installed as part of the pipeline? Then we would not have to include it in the repo.

af-rsa commented:
> Could the Maven Wrapper also be downloaded and installed as part of the pipeline? Then we would not have to include it in the repo.

It could be, and Wlad had it done it so, but I'd prefer to use wrapper for following reasons:

1) it is added by default by actuator
2) With it you don't need to install maven locally, and all the devs can ensure that they use the same maven version
3) You don't need to separately download maven or use maven docker image in the pipeline.

wlad reviewed (approved):
Looks good overall. @af-rsa please check if you want to update something based on my comments

wlad commented:
@af-rsa consider using image `cimg/openjdk:11.0.8`
![image](https://user-images.githubusercontent.com/64201310/97746487-ae11be80-1aea-11eb-956b-994fb9258594.png)

Commit:
added jsons for imunization "completed" and "not done"
added location of the jsons and robot file for imunization to the suite setting file
added immunization robot
* keyword for validate
* keyword for create
* keywords for create
added 01_create in Immunization folder with Testcases
* create History of vaccination - immunization completed
* create History of vaccination - immunization not done
added __init__.robot to folder immnuization

Commit:
Merge branch 'develop' into feature/NUM-511_integration_test_Vaccination

af-pwo committed:
[FIX] small counter typo
